undertow/src/main/java/org/wildfly/extension/undertow/handlers/ReverseProxyHandler.java

```diff
@@ -95,8 +96,8 @@
     public static final AttributeDefinition CONNECTION_IDLE_TIMEOUT = new SimpleAttributeDefinitionBuilder(Constants.CONNECTION_IDLE_TIMEOUT, ModelType.INT)
             .setRequired(false)
             .setAllowExpression(true)
-            .setDefaultValue(new ModelNode(60L))
-            .setMeasurementUnit(MeasurementUnit.SECONDS)
+            .setDefaultValue(new ModelNode(60000))
```

Comment:
We need a transformer to deal with this one. If this isn't configured a server running the current version will use 60000 and one running a previous version will us 60. The servers should be consistent, so the transformer would explicitly tell the legacy server to use 60000.

Unfortunately "Attribute has a different default value" at https://docs.wildfly.org/19.1/Extending_WildFly.html#common-transformation-use-cases doesn't explicitly illustrate how to do this